docs/reference/aggregations/bucket/terms-aggregation.asciidoc

```diff
@@ -218,24 +223,14 @@ instead. Due to the way the `terms` aggregation
 <<search-aggregations-bucket-terms-aggregation-shard-size,gets terms from
 shards>>, sorting by ascending doc count often produces inaccurate results.
 
-If you really must, this is how you sort on doc count ascending:
 
-[source,console,id=terms-aggregation-count-example]
---------------------------------------------------
-GET /_search
-{
-  "aggs": {
-    "genres": {
-      "terms": {
-        "field": "genre",
-        "order": { "_count": "asc" }
-      }
-    }
-  }
-}
---------------------------------------------------
+===== Ordering by the term value
+In this case, the buckets are ordered by the actual term values, e.g.
+Lexicographically for keywords or numerically for numbers.  This sorting is
+safe in both ascending and descending directions, and will always produce
+correct results.
```

Comment:
```suggestion
In this case, the buckets are ordered by the actual term values, such as
lexicographic order for keywords or numerically for numbers. This sorting is
safe in both ascending and descending directions, and produces accurate
results.
```

Reply:
We typically want to avoid phrases that include "always", which might not always be true 💡 . Instead, I think we can say that this sorting "produces accurate results".

Reply:
Nik and I spent like 20 minutes proving it before I typed "always", but yeah, as Knuth said "be careful, I've only proven it correct, I haven't actually tested it".